tools/video_tools/run-tesseract.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
+
+# Python imports
+import pytesseract
+import os
+import json
+import sys
+import time
+import subprocess
+import shlex
+import pprint
+import shutil
+# Python imports
+try:
+    from PIL import Image
+except ImportError:
+    import Image
+
+from datetime import datetime
+from decimal import Decimal
+from pytesseract import Output
+
+def main():
+	(input_file, output_name) = sys.argv[1:3]
+	os.mkdir(input_file[:-4])
+	dateTimeObj = datetime.now()
+
+	#ffmpeg part starts here
```

Comment:
could you add a comment about what the ffmpeg is doing here (something like "generate jpg files per frame")?

Reply:
done